pkg/processor/sumologicschemaprocessor/config.go

```diff
@@ -36,6 +37,7 @@ const (
 	defaultAddCloudNamespace           = true
 	defaultTranslateAttributes         = true
 	defaultTranslateTelegrafAttributes = true
+	defaultTranlateDockerMetrics       = true
```

Comment:
This causes this change to be a breaking one, how careful are we going to be with that?

Reply:
Lets make it optional and discuss next week

Reply:
IMHO, it should be enable by default and it is rather lower severity

Reply:
I set it to "false". I'll add this topic to Monday meeting's agenda